content/en/platform/corda/4.8/open-source/node-docker-deployments.md

```diff
@@ -0,0 +1,194 @@
+---
+aliases:
+- /head/node-docker-deployments.html
+- /HEAD/node-docker-deployment.html
+- /node-docker-deployments.html
+date: '2020-04-07T12:00:00Z'
+menu:
+  corda-os-4-8:
+    identifier: corda-os-4-8-node-docker-deployments
+    parent: corda-os-4-8-corda-nodes-index
+    weight: 1150
+tags:
+- node
+- docker
+title: Docker deployments
+---
+
+# Docker deployments
+
+This page contains a repository of example manual node configuration operations using Docker. Before executing any of these commands,
```

Comment:
Can you have a look at this intro and economise a bit? There are a few lines here - for more information on this... this page contains... where you could be more ruthless. 

This is a style issue, but it will help the document I think.